.github/workflows/scripts/scanner-update-nvd.py

```diff
@@ -59,11 +59,24 @@ def fetch_data(start, end):
 
             with request.urlopen(req, timeout=45) as response:
                 data = json.loads(response.read().decode())
+
             total = data['totalResults']
             logging.info(f"Fetched page at index {index} (out of {total} total items)")
-            yield from data['vulnerabilities']
+
+            # Transform the data during fetching
+            for item in data['vulnerabilities']:
+                cve_feed = item['cve']
+                transformed_data = {
+                    "id": cve_feed['id'],
+                    "published": cve_feed['published'],
+                    "lastModified": cve_feed['lastModified'],
+                    "descriptions": cve_feed.get('descriptions', []),
+                    "metrics": cve_feed.get('metrics', {})
+                }
+                yield {"cve": transformed_data}
```

Comment:
I think it's OK to put the whole CVE in the bundle. The size will increase when fetching in the updater exporter, but less work if we need additional information from NVD (we would only have to change the NVD feed script). Notice [the enricher already filters the necessary fields](https://github.com/stackrox/stackrox/blob/master/scanner/enricher/nvd/nvd.go#L304).